src/lein_monolith/config.clj

```diff
@@ -38,6 +38,14 @@
           next-files)))))
 
 
+(defn- add-raw-to-meta
+  "Attaches the raw monolith project as metadata to an initialized project.
+  It's necessary to have a reference to the raw project so unprocessed values
+  can be inherited with inherit-raw and inherit-leaky-raw."
+  [raw monolith]
```

Comment:
Idiom: this argument ordering would be less surprising if it matched `vary-meta`, with the base data structure first.